External/Plugins/HaXeContext/PluginMain.cs

```diff
@@ -172,19 +172,19 @@ public void HandleEvent(Object sender, NotifyEvent e, HandlingPriority priority)
                     break;
                 case EventType.Trace:
                     if (settingObject.DisableLibInstallation) return;
+                    var patterns = new[]
+                    {
+                        "Library \\s*(?<name>[^ ]+)\\s*?(\\s*version (?<version>[^ ]+))?",
+                        "Could not find haxelib\\s*(?<name>\"[^ ]+\")?(\\s*version \"(?<version>[^ ]+)\")?"//openfl project
+                    };
```

Comment:
Why not making the regular expressions compiled instead of having them run for every trace message?